polkadot/xcm/xcm-executor/src/lib.rs

```diff
@@ -983,18 +983,6 @@ impl<Config: config::Config> XcmExecutor<Config> {
 				);
 
 				let weight = message_call.get_dispatch_info().call_weight;
-
-				if !weight.all_lte(require_weight_at_most) {
-					tracing::trace!(
-						target: "xcm::process_instruction::transact",
-						%weight,
-						%require_weight_at_most,
-						"Max weight bigger than require at most",
-					);
-
-					return Err(XcmError::MaxWeightInvalid)
```

Comment:
haha, now we could remove `MaxWeightInvalid` from the xcm spec - related to the https://github.com/paritytech/polkadot-sdk/issues/6199
(no need to do anything here, just a comment :))